steamworks_unix.go

```diff
@@ -303,6 +303,20 @@ func (s steamFriends) GetPersonaName() string {
 	return C.GoString(C.uintptrToChar(C.uintptr_t(v)))
 }
 
+func (s steamFriends) SetRichPresence(key, value string) bool {
+	ckey := append([]byte(key), 0)
+	defer runtime.KeepAlive(ckey)
```

Comment:
Let's use C.CString and C.free for consistency for UNIX.

Reply:
Thanks, I've addressed this now and re-tested with latest push